spec/features/user/user_cant_see_admin_pages_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+feature 'When visiting admin pages' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.